samples/boards/96b_argonkey/sensors/README.rst

```diff
@@ -49,27 +55,40 @@ Building and Running
 Sample Output
 =============
 
+A USB to TTL 1V8 serial cable may be attached to the low speed connector on
+the back of the board on P3.5 (TX) and P3.7 (RX). User may use a simple
+terminal emulator, such as minicom, to capture the output.
+
```

Comment:
The ``.. code-block`` directive below should not be indented by a space.  Can you also fix the indent on the code-block on line 23.